File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/GruppoRaccolta.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class GruppoRaccolta: baseClass.BaseCodeDescr
    {
        public IntervalloTemporale validita { get; set; }

        public GruppoRaccolta(int code, string descr): base(code, descr)
        {
        }

        public GruppoRaccolta() { }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/GruppoRaccolta.ts
import {BaseCodeDescr} from '../baseClass/baseCodeDescr';
import {IntervalloTemporale} from '../anagrafiche/IntervalloTemporale';

export class GruppoRaccolta extends BaseCodeDescr {

    validita: IntervalloTemporale;
    constructor(codice: number) {
        super(codice);
    }
}
export class ScriviGruppoRaccolta {
    public gruppoRaccolta: GruppoRaccolta;

    public tipoOperazione: number;
}
